multi_block_core.dic

```diff
@@ -56,6 +56,49 @@ save_MULTIBLOCK_CORE
 
 save_
 
+save_AUDIT_DATASET
+
+    _definition.id                AUDIT_DATASET
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2025-04-14
+    _description.text
+;
+    The CATEGORY of data items used for linking data blocks to aid in grouping
+    blocks which contain information which are interrelated.
+
+    Data blocks which belong together are identified with the same unique id.
+;
+    _name.category_id             AUDIT
+    _name.object_id               AUDIT_DATASET
+    _category_key.name            '_audit_dataset.id'
+
+save_
+
+save_audit_dataset.id
```

Comment:
It might be worthwhile to clarify the intended valid scope of the ID, i.e.:
- Is it valid only within a single file/container or across multiple files? (Probably the latter?)
- Should the recommendation be to select an universally unique ID (e.g. by using something like UUIDs). This could prove useful when processing data in a stream/pipeline.